pokerth.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2014 PokerTH GbR <webmaster@pokerth.net> -->
+<component type="desktop-application">
+ <id type="desktop">pokerth.desktop</id>
+ <metadata_license>CC-BY-3.0</metadata_license>
+ <project_license>AGPL-3.0</project_license>
+ <name>PokerTH</name>
+ <developer_name>PokerTH GbR</developer_name>
+ <summary>Poker game with single- and multiplayer support</summary>
+ <description>
+	<p>PokerTH is a poker game for the popular Texas Hold'em Poker variant. You can play singleplayer games with up to 9 computer-opponents or meet other PokerTH players around the world via internet games. Also multiplayer games in your LAN are possible with the network game mode. If you like to change the look and feel of the PokerTH gametable or the card deck just visit the styles gallery and choose your favourite style. To compare your poker skills with the other PokerTH players you can check your game results and rankings on our website.</p>
+ </description>
+ <screenshots>
+  <screenshot type="default">
+    <image>https://upload.wikimedia.org/wikipedia/commons/b/bd/PokerTH.png</image>
+  </screenshot>
+ </screenshots>
+ <url type="homepage">https://www.pokerth.net</url>
+ <url type="bugtracker">https://github.com/pokerth/pokerth/issues</url>
+ <url type="faq">https://www.pokerth.net/app.php/help/faq</url>
+ <url type="donation">https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&amp;hosted_button_id=FU66XN5FMMBNQ</url>
```

Comment:
Donation link seems to be not working for me.

Reply:
https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=FU66XN5FMMBNQ